lua/lspsaga/definition.lua

```diff
@@ -256,8 +258,9 @@ function def:goto_definition(method)
     if vim.bo.modified and current_buffer == jump_destination then
       vim.cmd('write!')
     end
-
-    api.nvim_command('edit ' .. jump_destination)
```

Comment:
there maybe no necessary in goto_definition because it check before . see above .

Reply:
You mean this? But that doesn't reduce loc